components/Models.tsx

```diff
@@ -404,7 +404,7 @@ export default function Models() {
             New model
           </Button>
 
-          {schema.database !== "sqlite" && (
+          {schema.database !== "sqlite" && schema.database !== "sqlserver" && (
```

Comment:
If we want to check for more providers in the future, maybe this syntax is a bit more maintainable.

```
!["sqlite", "sqlserver"].includes(schema.database)
```